README.md

```diff
@@ -212,6 +212,8 @@ the above as root. For example, in Ubuntu:
 Now you can use the `mypy` program just as above.  In case of trouble
 see "Troubleshooting" above.
 
+# NOTE: Installing with sudo can be a security risk, please try with flag `--user` first.
```

Comment:
I would change this from a main header to quoted text, which is stylistically in line with the docs.
```suggestion
> NOTE: Installing with sudo can be a security risk, please try with flag `--user` first.
```